Program.fs

```diff
@@ -148,25 +137,34 @@ module MainModule =
                         tally = tally
                     }
             }
-        let! response = buildMessage Http.HttpMethod.Post url (Some header) (Some message) |> makeRequest
+        let! response =
+            buildMessage Http.HttpMethod.Post url (Some header) (Some message)
+            |> makeRequest
         
         return response |> deserialize<SwuResponse>
     }
 
     [<EntryPoint>]
     let main argv =
-        let mayFifth2017 = 1493614800000L
         let since = midnightYesterday () |> toUnixTimestamp
         let until = midnight () |> toUnixTimestamp
         let protocol = if isLive then "https" else "http"
         let url = sprintf "%s://%s/api/v1/orders/portraits/artist-tally?since=%i&until=%i" protocol apiDomain since until
-        let summaryResponse = buildMessage Http.HttpMethod.Get url None None |> makeRequest |> Async.RunSynchronously |> deserialize
-
-        if summaryResponse.summary.Count = 0 then 
-            printfn "Tally response contained an empty summary. Was the `since` parameter (%i) incorrect?" since
-        else 
-            for kvp in summaryResponse.summary do printfn "%s: %i portraits" kvp.Key kvp.Value
-
-        let emailResponse = summaryResponse.summary |> convertResponseToTally |> sendEmailMessage |> Async.RunSynchronously
+        let summaryResponse =
+            buildMessage Http.HttpMethod.Get url None None
+            |> makeRequest
+            |> Async.RunSynchronously
+            |> deserialize
+
+        match summaryResponse.summary.Count with
```

Comment:
For arbitrary boolean conditions, `if...then...else` is okay, but when you check for very specific things like concrete number values, consider using pattern matching, as gives the individual cases that are being evaluated a more equal visual representation.